src/middlewares/createInsightsMiddleware.ts

```diff
@@ -38,26 +38,35 @@ export const createInsightsMiddleware: CreateInsightsMiddleware = props => {
     _insightsClient === null ? (noop as InsightsClient) : _insightsClient;
```

Comment:
If we type it correctly here, we shouldn't have to cast `insightsClient` every time.

Reply:
Do you mean regarding `(insightsClient as any).queue` right?
https://github.com/algolia/instantsearch.js/pull/4575/commits/5ddedeec8806d997372d27de340f1194cce2e174